demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1445,6 +1446,21 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
     return tiles;
   }
 
+  setUpVideoTileElementResizer(): void {
+    for (let i = 0; i <= 16; i++) {
+      const videoElem = document.getElementById(`video-${i}`) as HTMLVideoElement;
+      videoElem.onresize = () => {
+        this.log(`video-${i} intrinsic resolution ${videoElem.videoWidth}x${videoElem.videoHeight}`);
```

Comment:
Do you want to log every resize? My concern is that any time the viewport is resized or when another video is added or removed and affects the layout, then this function will fire for up to 16 video elements. It might be worth only logging if the video dimensions matches the portrait dimensions you're looking for?